cfpb/cfgov-refresh
# Add helper text to HMDA data download form fields

We need subtitles under three of the HMDA data download fields. Instead of creating a custom form field type with a custom widget, it's easier to conditionally add the text in the template.

## Changes

- Adds subtitles to several of the HMDA data download fields.
- Fixes the institution links to point to zips instead of CSVs.
- Adds aria labels to institution files.
- Couple language corrections.

## Testing

1. Create a `Hmda historic data page` page.
1. Optionally add any content to the page.
1. Publish the page and you should see the subtitles under three of the form fields.

## Screenshots

<img width="692" alt="Screen Shot 2019-04-01 at 6 26 52 PM" src="https://user-images.githubusercontent.com/1060248/55363707-edf66700-54ab-11e9-9088-d505fc4aea88.png">

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
